packages/grid/_modules_/grid/models/colDef/gridColDef.ts

```diff
@@ -160,13 +160,13 @@ export interface GridColDef {
   disableExport?: boolean;
 }
 
-export interface GridColumnProp extends Omit<GridColDef, 'filterOperators'> {
-  filterOperators?: GridFilterOperator[] | string;
-}
-
 export type GridColumns = GridColDef[];
 export type GridColTypeDef = Omit<GridColDef, 'field'> & { extendType?: GridNativeColTypes };
 
+export interface GridStateColDef extends GridColDef {
```

Comment:
Why do we need to extend here, can't we just add it directly to the `GridColDef`, if it is going to be publicly available?

Reply:
It allows to differenciate the input columns and the state columns
If I add it to `GridColDef`, then the users could think they can provid a `computedWidth`.


`GridColDef` = the prop `columns` given to `DataGrid` and `XGrid`
`GridStateColDef` = the columns stored in the state and returned in callbacks.

The only difference is currenctly the `computedWidth` but I think their will be more preprocessings and additions in the future.